include/zephyr/linker/linker-defs.h

```diff
@@ -133,6 +133,7 @@ extern char _flash_used[];
 /* datas, bss, noinit */
 extern char _image_ram_start[];
 extern char _image_ram_end[];
+extern char _image_ram_used[];
```

Comment:
```suggestion
extern char _image_ram_size[];
```
to be aligned with other definitions.

Reply:
which, based on the commit message was also the original intention, as the commit message says:
> Add _image_ram_size symbol to linker definitions. This symbol
> corresponds to the total RAM used by the image.

Reply:
Symbol name is updated to `_image_ram_size`